lib/generators/linked_data_fragments/templates/config/ldf.yml

```diff
@@ -0,0 +1,21 @@
+development:
+  uri_endpoint: 'http://localhost:3000/{?subject}'
```

Comment:
This now works fairly wonky. First is that it is no longer at that route since it default maps itself to "/ldf" as an engine. Furthermore, if you change this value to something like 'http://localhost:3000/test/{?subject}' , it seems to break for me.